modules/hal_wch/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
-if(CONFIG_SOC_CH32V003 OR CONFIG_SOC_SERIES_QINGKE_V4C)
+if(CONFIG_SOC_CH32V003 OR CONFIG_SOC_SERIES_QINGKE_V4)
```

Comment:
Wouldn't it be better to put just `CONFIG_SOC_FAMILY_CH32V` ? I don't see why a ch32v soc wouldn't use the HAL.